syndesis/src/i18n/index.ts

```diff
@@ -22,17 +23,11 @@ const options = {
   },
 
   defaultNS: 'shared',
-  fallbackLng: 'it',
+  fallbackLng: process.env.NODE_ENV === 'production' ? 'en' : 'it',
+  fallbackNS: ['shared'],
   keySeparator: false, // we do not use keys in form messages.welcome
-  ns: ['shared'],
+  ns: ['shared', 'app', 'dashboard'],
+} as InitOptions;
```

Comment:
I had to add the `as InitOptions` statement so that `keySeparator` would work. Not sure why it worked before. Maybe it is due to upgrading the version.

Reply:
That's because before there was no proper typing for this export. This is correct.